fjord/feedback/views.py

```diff
@@ -45,8 +46,15 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
-    return render(request, template)
+    user_opinion = Response.objects.get(id=request.session['opinion_id'])
+    length = len(user_opinion.description.split())
```

Comment:
Instead of calling this "length", I'd use something more descriptive like "word_count".